css/style.css

```diff
@@ -1,19 +1,210 @@
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
-}
+/* phone size quary */
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+@media only screen and (max-width: 600px){ 
+    .body {
+        font-family: 'Roboto', sans-serif;
+        -webkit-font-smoothing: antialiased;
+        display: inline-block;
+        grid-template-rows: auto;
+        background-color: rgb(27, 196, 139);
+        }
+        
+        label {
+            display: block;
+            cursor: pointer;
+            width: 30px;
+            font-size: 20px;
+            float: right;
+            
+        }
+        .navigation{
+            display: grid;
+            grid-template-columns: 90% 10%;
+        }
 
+        .Nav a:hover, button:hover {
+            color: rgb(59, 25, 12);
+            font-style: italic;
+            font-weight: bolder;
+        }
+        #toggle{
+            visibility: hidden;
+        }
 
+        #toggle, .toggle{
+            grid-column: 2;
+            grid-row: 1;
+            position:relative;
+            
+        }
+        .Nav {
+            text-align: center;
+            width: 100%;
+            display: none;
+        }
+        .Nav a {
+            display: block;
+            border-bottom: 1px solid #EAEAEB;
+            padding: 10px;
+         
+        }
+        #toggle:checked + .Nav {
+            display: block;
+            text-align: center;
+        }
+
+ .logo {
+       width: 30px;
+       height: 30px;
+       padding-left: 15px;
+       position: relative;
+        }
+    
+    .backgroundimage {
+        width: 100%;
+        height: 50%;
+    }
+    .header {
+        width: 99%;
+        flex-direction: column;
+    }
+    .maintitle {
+       position: relative;
+       top: -100px;
+       text-align: center;
+       color: lavenderblush;
+       font-size: 20px;
+        }
+ .main {
```

Comment:
Just be careful about spaces between rules and general indentation. All the selectors should be aligned here :)